src/utils/env.ts

```diff
@@ -142,19 +142,24 @@ export const ENV = {
     return castStringEnv('AUTH_USER_DEFAULT_ROLE', 'user');
   },
   get AUTH_USER_DEFAULT_ALLOWED_ROLES() {
-    return castStringArrayEnv('AUTH_USER_DEFAULT_ALLOWED_ROLES', [
+    const allowedRoles = [
       'me',
```

Comment:
In a future major release, let's consider if we should remove this hardcoded `me` role.